[2025-02-12T00:03:59.406+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T00:03:59.447+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_for_dbt_raw.Upload_data_to_s3 scheduled__2025-02-08T00:00:00+00:00 [queued]>
[2025-02-12T00:03:59.482+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_for_dbt_raw.Upload_data_to_s3 scheduled__2025-02-08T00:00:00+00:00 [queued]>
[2025-02-12T00:03:59.491+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-12T00:03:59.585+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): Upload_data_to_s3> on 2025-02-08 00:00:00+00:00
[2025-02-12T00:03:59.598+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=351) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-12T00:03:59.613+0000] {standard_task_runner.py:72} INFO - Started process 363 to run task
[2025-02-12T00:03:59.616+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_for_dbt_raw', 'Upload_data_to_s3', 'scheduled__2025-02-08T00:00:00+00:00', '--job-id', '151', '--raw', '--subdir', 'DAGS_FOLDER/dbt_raw_dag.py', '--cfg-path', '/tmp/tmpdwrj18rf']
[2025-02-12T00:03:59.618+0000] {standard_task_runner.py:105} INFO - Job 151: Subtask Upload_data_to_s3
[2025-02-12T00:03:59.755+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_for_dbt_raw.Upload_data_to_s3 scheduled__2025-02-08T00:00:00+00:00 [running]> on host ecdf8258250e
[2025-02-12T00:04:00.054+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_for_dbt_raw' AIRFLOW_CTX_TASK_ID='Upload_data_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-02-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-08T00:00:00+00:00'
[2025-02-12T00:04:00.057+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-12T00:08:05.769+0000] {logging_mixin.py:190} INFO - uploaded file sub.txt
[2025-02-12T00:09:21.503+0000] {logging_mixin.py:190} INFO - uploaded file pre.txt
[2025-02-12T00:13:30.566+0000] {logging_mixin.py:190} INFO - uploaded file num.txt
[2025-02-12T00:13:46.220+0000] {logging_mixin.py:190} INFO - uploaded file tag.txt
[2025-02-12T00:13:47.130+0000] {logging_mixin.py:190} INFO - uploaded file readme.htm
[2025-02-12T00:13:47.309+0000] {python.py:240} INFO - Done. Returned value was: success
[2025-02-12T00:13:47.758+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-12T00:13:47.767+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_for_dbt_raw, task_id=Upload_data_to_s3, run_id=scheduled__2025-02-08T00:00:00+00:00, execution_date=20250208T000000, start_date=20250212T000359, end_date=20250212T001347
[2025-02-12T00:13:47.993+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T00:13:48.280+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-12T00:13:48.286+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
